app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
We tried to normalize expected provider exception to MiqException so that the automate does not need to catch exceptions per provider type. Even though here whether we convert the exception or not does make big difference and the provisioning will eventually fail, there are cases that we do need to catch specific MiqException.